stable/rails/values.yaml

```diff
@@ -24,7 +24,15 @@ migrations:
   commands:
   - "bundle exec rake db:migrate"
 
-ingress:
+ingress_private:
```

Comment:
Instead of 2 ingress types - what if we use array for ingress. As an example:
```
ingress:
- type: public
   host: myhost1.local
- type: public
   host: myhost1.local
- type: private
   host: myhost2.local
```